### DISCUSSION TITLE
Add dynamic pagetree / child pages / children to a page

### DISCUSSION THREAD

**Discussion by mzeecedric** [outline/outline#8076] [Ideas]
One thing that I miss is the possibility to add a pagetree to the content of a page. 
What I expect is to display a list of children of 1) the current page (as default) and optionally 2) the children of an other page that can be specified. In the first place this could be a simple list of the first descendants, optionally displaying more descendant levels. This is basically the same as "Documents" at the bottom of a page or the sidebar navigation. 

Once we have tags ( #765  +1 ) it would be also nice to list all pages with a certain tag in the same way in the content.

**Comment by foxxyz** [comment]
I'll add my two cents to this... Coming from Confluence where you can add a subtree or list of direct children, it looks strange to be in a parent page and just see an empty page.

Some people add "Look in the sidebar for child pages" text or even add the links to child pages manually to compensate.

I guess an alternative would be to hoist the "Documents" section to the top if the parent page has no content? Either way this feels like   something that's missing.

Thanks! 🙏🏽

**Reply by tommoor** [reply]
> I guess an alternative would be to hoist the "Documents" section to the top if the parent page has no content? Either way this feels like something that's missing.

Yea, this would be ideal - and is actually the current behavior if you have the setting for a separate "Editing mode" enabled. The reason it doesn't happen with inline editing is that it would make it difficult to discern where to type?

**Reply by foxxyz** [reply]
The inline editing still shows the placeholder for `Type '/' to insert, or start writing; so it seems pretty clear to me?

I didn't realize it pulls it up further on the separate editing mode (I've had it on inline editing since the beginning), but I guess I don't understand why the vertical space is so much bigger.

<img width="1215" height="697" alt="Screenshot 2026-03-31 at 16 26 29" src="https://github.com/user-attachments/assets/12d3c10f-a046-4db2-86a7-95afd57dd2e6" />

It feels big and disconnected. Does it have to be?

**Reply by tommoor** [reply]
It doesn't _have_ to be, we could try it pulled up in both scenarios, see if any confusion occurs

**Reply by foxxyz** [reply]
Thanks for the consideration @tommoor !
